OSDOCS-2407 - Adding etcd encryption section (#39955)
openshift/openshift-docs: modules/rosa-sdpolicy-security.adoc

[id="rosa-sdpolicy-network-security_{context}"]
== Network security
With {product-title}, AWS provides a standard DDoS protection on all load balancers, called AWS Shield. This provides 95% protection against most commonly used level 3 and 4 attacks on all the public facing load balancers used for {product-title}. A 10-second timeout is added for HTTP requests coming to the `haproxy` router to receive a response or the connection is closed to provide additional protection.

[id="rosa-sdpolicy-etcd-encryption_{context}"]
== etcd encryption

> In {product-title}, the control plane storage is encrypted at rest by default and this includes encryption of the etcd volumes. This storage-level encryption is provided through the storage layer of the cloud provider. In addition to this, you can opt to enable etcd encryption, which encrypts the key values in etcd state. The etcd encryption feature is not enabled by default and it can be enabled only at cluster installation time.

Reviewer: ```suggestion
In {product-title}, the control plane storage is encrypted at rest by default and this includes encryption of the etcd volumes. This storage-level encryption is provided through the storage layer of the cloud provider. You can also enable etcd encryption, which encrypts the key values in etcd state. The etcd encryption feature is not enabled by default and it can be enabled only at cluster installation time.
```

Author: @jeana-redhat thank you for your review and for this suggestion. This is now resolved.

Author: I have resolved this in both files.

[IMPORTANT]
====

> By enabling etcd encryption for the key values in etcd state, you might incur a performance overhead of up to 20%. Red Hat recommends that you enable etcd encryption only if you specifically require this for your use case, in addition to the default storage-level encryption.

Reviewer: ```suggestion
By enabling etcd encryption for the key values in etcd state, you might incur a performance overhead of up to 20%. Red Hat only recommends that you enable etcd encryption in addition to the default storage-level encryption if you specifically require this for your use case.
```

Author: Great suggestion, thank you. Resolved.

Author: I have resolved this in both files.
